dubbo-config/dubbo-config-api/pom.xml

```diff
@@ -178,46 +132,13 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
```

Comment:
Depedency of metadata-api will cause cyclic reference, and related tests needs to move.